AaxDecrypter/AaxcDownloadConverter.cs

```diff
@@ -211,13 +211,27 @@ public bool Step4_RestoreMetadata()
 
         public bool Step5_CreateCue()
         {
-            File.WriteAllText(PathLib.ReplaceExtension(outputFileName, ".cue"), Cue.CreateContents(Path.GetFileName(outputFileName), downloadLicense.ChapterInfo));
+            try
+            {
+                File.WriteAllText(PathLib.ReplaceExtension(outputFileName, ".cue"), Cue.CreateContents(Path.GetFileName(outputFileName), downloadLicense.ChapterInfo));
+            }
+            catch (Exception ex)
+            {
+                //TODO Add log entry
```

Comment:
If you're not ready to write out the serilog call then better to not have a try/catch swallow it so that at least the next layer up can catch it.

Reply:
OK, I added an error log.